cmd/e2e_test.go

```diff
@@ -1,15 +1,28 @@
 package cmd
 
 import (
+	"context"
 	"testing"
 
 	"github.com/onsi/ginkgo/v2"
 	"github.com/onsi/gomega"
+	"github.com/redhat-appstudio/e2e-tests/pkg/client"
 	_ "github.com/redhat-appstudio/e2e-tests/pkg/tests/common"
 	_ "github.com/redhat-appstudio/e2e-tests/pkg/tests/has"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
+	KubeClient, err := client.NewK8SClient()
+	if err != nil {
+		panic(err)
+	}
+	secret, err := KubeClient.KubeInterface().CoreV1().Secrets("application-service").Get(context.TODO(), "has-github-token", metav1.GetOptions{})
+	if err != nil {
+		panic(err)
```

Comment:
Dont panic the tests, use gomega to check the errors.

Also print errors with klog. Example:https://github.com/redhat-appstudio/e2e-tests/pull/16/files#diff-474bfa1e842951b4a90fd7715d82bef58de35d5ce530efa7c2500e3e304dec0dR43